docs/content/examples/tutorials/basic-installation.md

```diff
@@ -2,25 +2,23 @@
 title: 'Tutorials | Basic Installation'
 ---
 
-## Building a Simple Mail-Server
+## Setting up a Simple Mail-Server
 
-!!! warning
-    Adding the docker network's gateway to the list of trusted hosts, e.g. using the `network` or `connected-networks` option, can create an [**open relay**](https://en.wikipedia.org/wiki/Open_mail_relay), for instance [if IPv6 is enabled on the host machine but not in Docker][github-issue-1405-comment].
+This is a community contributed guide. Please let us know via a Github Issue if you're having any difficulty following the guide so that we can update it.
 
-We are going to use this docker based mailserver:
+This guide is focused on only using [SMTP ports (not POP3 and IMAP)][docs-ports] with the intent to send received mail to another MTA service such as _Gmail_. It is not intended to have a MUA client (_eg: Thunderbird_) to retrieve mail directly from `docker-mailserver` via POP3/IMAP.
 
-- First create a directory for `docker-mailserver` to store data in, and get the `setup.sh` script:
+In this setup `docker-mailserver` is not intended to receive email externally, so no anti-spam or anti-virus software is needed, making the service lighter to run.
 
-    ```sh
-    mkdir -p /var/ds/mail.example.com
-    cd /var/ds/mail.example.com/
+!!! warning "Open Relays"
 
-    curl -o setup.sh \
-        https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/setup.sh
-    chmod a+x ./setup.sh
-    ```
+    Adding the docker network's gateway to the list of trusted hosts (_eg: using the `network` or `connected-networks` option_), can create an [**open relay**](https://en.wikipedia.org/wiki/Open_mail_relay). For instance [if IPv6 is enabled on the host machine, but not in Docker][github-issue-1405-comment].
+
+1. If you're running a version of `docker-mailserver` earlier than v10.2, [you'll need to get `setup.sh`][docs-setup-script]. Otherwise you can substitute `./setup.sh <command>` with `docker exec mailserver setup <command>`.
 
-- Create the file `docker-compose.yml` with a content like this:
+2. Pull the docker image: `docker pull docker.io/mailserver/docker-mailserver:latest`.
```

Comment:
We should advice to use versioned tags instead of :latest.
Or at least mention, that using :latest can lead to unwanted updates.

Reply:
> Or at least mention, that using :latest can lead to unwanted updates.

It's usually the opposite?

When relying on the `:latest` tag, you may not get the actual latest version of that tag if a local one is found. This is different with `pull` command which will fetch the latest, but in this case for the guide, I think it would be assumed that's intended? (as in starting fresh)

We normalized on `:latest` because some docs content was not being maintained as actively, the 10.1 version of docs reference a tag of v7.x IIRC for `docker-mailserver` on one page. The warnings for relying on `:latest` in production is fairly widespread in the Docker communities last I knew?

I don't mind making a change on this if more than one maintainer would like to, however I don't think `:latest` can be considered as a immutable tag. If a user expects to avoid unwanted updates, they should use a tag to pin the release. `:latest` is default right? Would you prefer this tag was just omitted and used implicitly?

Reply:
You are absolutely right, in all what you said. Users should know about :latest tag and what this means in terms of updates.
It's good the docs were normalized. But just in that case, I thought, people tend to be lazy. And as this is a basic example, I bet, many will just copy/paste without knowing exactly what each line means. Especially when they are new to Docker.
Also me 😄: People shouldn't start running this mail-server, when they don't have basic knowledge about Docker/SMTP.

> This is different with pull command

It's not only the pull command. I am personally against auto-updates, where images are independently updated and containers restarted. However we give a manual [here](https://pullrequest-2210--dms-doc-previews.netlify.app/config/advanced/maintenance/update-and-cleanup/) to do so.
So using the basic example + auto updates, can lead to unexpected automated failures in the future.

Conclusion: If others also approve this, I am also fine to leave it as it is 👍

Reply:
Have you noticed it as a recurring problem that occurs in issues raised for the project? If so, that'd be a good reason to ensure the user is better informed.

On the TLS page I have been expanding on details a little more as I think most readers are lacking knowledge and awareness in that area.

To be fair, we have been having issues raised recently including duplicates from users that aren't bothering to properly read the README, and then deciding `:edge` tag resolves their problem without thinking that through..